pkg/portal/wire_gen.go

```diff
@@ -296,14 +296,19 @@ func newGraphQLHandler(p *deps.RequestProvider) http.Handler {
 	}
 	remoteIP := deps.ProvideRemoteIP(request, trustProxy)
 	userAgentString := deps.ProvideUserAgentString(request)
+	writeHandle := auditdb.NewWriteHandle(context, pool, databaseEnvironmentConfig, auditDatabaseCredentials, logFactory)
+	writeSQLExecutor := auditdb.NewWriteSQLExecutor(context, writeHandle)
```

Comment:
@louischan-oursky I think it is possible that the app's audit db is not the same as the portal audit db. And now `auditDatabaseCredentials` comes from environment config of portal but not from config of app, therefore we may write to wrong db if the app has another audit db configured?

Reply:
Actually the portal only reads ONE audit database from the environment variable. And it is assumed (for now) the app audit db is the same as the portal audit db.